ifStatement.js

```diff
@@ -59,6 +73,13 @@ Steps:
     - If the user is younger than 18, the program should calculate how many years are left until they turn 18 and print the message "You will be eligible to vote in X years", where X is the number of years left.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+let userage=18
+if (userage>=18)
+if(confirm("press" )) console.log(`you press ok `);
+else(console.log("Why did you waste my time then? 😀."))
```

Comment:
The block of code within the else statement should be wrapped with curly brackets {}, not parentheses. Like this:
else {console.log("Why did you waste my time then? 😀.") }

There's a missing calculation for the number of years left until the user turns 18 if they are younger than 18.